subcontracted_service/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Trigger procurement for service product
+=======================================
+
+This module adds the functionality of being able to automatically make a
+purchase order (through procurements) when a service product if sold
+to a customer.
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Just install it
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. configure you service product to check ``property_subcontracted_service``
+in product form if this product should trigger a procurement.
+
+#. add supplier in your product form
```

Comment:
Even though you added the hook, I think it would be good to explain how and where to set up the subcontracting rule for the company.